polytracker/taint_dag.py

```diff
@@ -389,11 +397,13 @@ def decode_node(self, label: int) -> TDNode:
             if v1 > v2:
                 return TDUnionNode(v1, v2, affects_cf)
             else:
+                if v == 0:
```

Comment:
nitpick: probably better to move this one nest up and do
```python
if v == 0:
...
elif v1 > v2:
...
else:
...
```

Reply:
My thinking is that this should be a rare case, that's why I didn't want to put it at the top. But maybe it should go there instead? Something like:
```python
if label == 0:
    return TDUntaintedNode()
````

Reply:
Yeah, if that 's possible, I think we should do that! Just to get the obvious cases out the way ASAP in the code,